### ISSUE TITLE
toast message doesn't show up in expo sdk 36 (react native 0.61)

### ISSUE THREAD

**Issue by andordavoti** [shx996/react-native-tiny-toast#13]
I used this in an expo project using expo sdk 35 (react native 0.59). When I upgraded to expo sdk 36 (react native 0.61) the toast messages no longer show up. No errors are logged or shown, the toast is just not visible.

**Comment by JCBsystem** [comment]
same for me

**Comment by andordavoti** [comment]
maybe this helps: https://github.com/expo/expo/issues/6473#issuecomment-565761931

**Comment by JCBsystem** [comment]
added 
 <RootSiblingParent>
App ..
</RootSiblingParent>
and now it works again

**Comment by andordavoti** [comment]
@JCBsystem where did you add that in your project? And where do you import RootSiblingParent from?

**Comment by JCBsystem** [comment]
my project is expo project 
add 
import { RootSiblingParent } from 'react-native-root-siblings'
 in app.js

then in render

```

return (
        <Provider store={store}>


            <RootSiblingParent>

              <AppNavigator />
            </RootSiblingParent>

  
        </Provider>
      );
```

**Comment by andordavoti** [comment]
@JCBsystem didn't work for me:/ the toast is still not showing up

**Comment by JCBsystem** [comment]
what version did you install of the react-native-root-siblings ? 
i use v4

and i also download tiny-toast locally and change it to same version 
also patched the componentWillReceiveProps to UNSAFE_componentWillReceiveProps

**Comment by andordavoti** [comment]
I also use react-native-root-siblings v4. Could you send me the tiny-toast as you have it modified? My email is andor.davoti@gmail.com

**Comment by serhiipalash** [comment]
This issue should be resolved in expo v36.0.1 https://github.com/expo/expo/issues/6473#issuecomment-566356579
